infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -219,11 +236,22 @@ public void sendWithBuffering(List<LogReplicationEntryMsg> dataToSend, String me
     /**
      * Resend the messages in the queue if they have timed out.
      */
-    public LogReplicationEntryMsg resend() {
+    public LogReplicationEntryMsg resend(boolean isLogEntry) {
         LogReplicationEntryMsg ack = null;
         boolean force = false;
+
         try {
             ack = processAcks();
+
+            if (isLogEntry && isBackpressureActive.get()) {
+                try {
+                    log.trace("Dropped ACKs from sink, resending log entry messages in some time...");
```

Comment:
we aren't dropping ACKs here, we are not getting it at all...so maybe reword this log statement?